docs/source/markdown/podman-create.1.md.in

```diff
@@ -766,6 +766,11 @@ Podman allocates unique ranges of UIDs and GIDs from the `containers` subordinat
 
 **keep-id**: creates a user namespace where the current rootless user's UID:GID are mapped to the same values in the container. This option is not allowed for containers created by the root user.
 
+  Valid `keep-id` options:
```

Comment:
Can you help out @edsantiago here and move this section into the options/subdir.

Reply:
added a new commit that moves the options to a separate file under `options/`